handlers.go

```diff
@@ -158,9 +170,9 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 				httpError(w, r, http.StatusInternalServerError, "encoding failed with error: %v", err)
 				return
 			}
-			// convert the input hash to hexadecimal for logging
-			hashlog = fmt.Sprintf("%X", input)
-
+			// the input is already a hash just convert it to hex
+			inputHash = asHex(input)
```

Comment:
perhaps this is just me, but I find the `asHex` more confusing than just calling `fmt.Printf` here.